homeassistant/components/discord/notify.py

```diff
@@ -1,7 +1,9 @@
 """Discord platform for notify component."""
 from __future__ import annotations
 
+from io import BytesIO
 import logging
+import ntpath
```

Comment:
I think we prefer `pathlib` when possible. Also ntpath is for windows so i'm having trouble reasoning about using that here...